sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -109,31 +109,33 @@ public final class CosmosAsyncClient implements Closeable {
         this.tokenCredential = builder.getTokenCredential();
         this.sessionCapturingOverride = builder.isSessionCapturingOverrideEnabled();
         this.enableTransportClientSharing = builder.isConnectionSharingAcrossClientsEnabled();
-        this.clientTelemetryConfig = builder.clientTelemetryConfig();
-        this.isSendClientTelemetryToServiceEnabled = ImplementationBridgeHelpers
+        ImplementationBridgeHelpers.CosmosClientTelemetryConfigHelper.CosmosClientTelemetryConfigAccessor
+            telemetryConfigAccessor = ImplementationBridgeHelpers
+            .CosmosClientTelemetryConfigHelper
+            .getCosmosClientTelemetryConfigAccessor();
+
+        CosmosClientTelemetryConfig effectiveTelemetryConfig = ImplementationBridgeHelpers
```

Comment:
we can use the telemetryConfigAccessor defined above

Reply:
Fixed